src/block_cache/block_io.c

```diff
@@ -689,6 +740,16 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *
     WT_STAT_SESSION_INCRV(session, bytes_write, mem_size);
     (void)__wt_atomic_add64(&S2C(session)->cache->bytes_written, mem_size);
 
+    if (dsk != NULL) {
```

Comment:
Does this work for the delta case? It looks like `dsk` and `delta` refer to two separate objects, either of which maybe become the `ip` that we pass to the block manager to write.

Reply:
That's a good catch, yes I agree we probably need to determine whether we want `dsk` or `delta`.